features/step_definitions/schools/onboarding_steps.rb

```diff
@@ -14,7 +14,7 @@
 Given "I have completed the DBS Requirements step, choosing No" do
   steps %(
     Given I am on the 'DBS requirements' page
-    And I choose 'No - Candidates will be accompanied at all times' from the 'Do you require candidates to have or get a DBS check?' radio buttons
+    And I choose 'No - Candidates will be accompanied at all times' from the 'Do candidates need to have or get a DBS check?' radio buttons
```

Comment:
This whole question needs re-designing, as per the content recommendations, so I think we should do the changes all in one go with dev help.